PWGHF/HFC/DataModel/CorrelationTables.h

```diff
@@ -200,6 +205,64 @@ DECLARE_SOA_TABLE(DplusHadronRecoInfo, "AOD", "DPLUSHRECOINFO", //! D+-Hadrons p
                   aod::hf_correlation_dplus_hadron::MD,
                   aod::hf_correlation_dplus_hadron::SignalStatus);
 
+// definition of columns and tables for Dstar-Hadron correlation pair
+using Dstars = HfCandDstar;
+using Dstar = Dstars::iterator;
```

Comment:
Why can you not use the original name?

Reply:
Because we did not have an iterator for HfCandDstar beforehand, also to make a difference that one is a full table another is an iterator, which did not have earlier. This helped me in adding Index column in ```DstarHadronPair`` table as 
```hf_correlation_dstar_hadron::DstarId```.

Reply:
Fair point, but that is not a reason to create an alias for an existing type. It's a reason to add `s` in the type name `HfCandDstar` -> `HfCandDstars` and add the iterator `using HfCandDstar = HfCandDstars::iterator;`.